File: springjpagroovycrud/src/main/java/com/knf/dev/demo/springjpagroovycrud/repository/UserRepository.java
package com.knf.dev.demo.springjpagroovycrud.repository;


import com.knf.dev.demo.springjpagroovycrud.model.User;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface UserRepository extends CrudRepository<User, Long> {
}

File: kotlin-spring-main/kotlinspringjpamustachecrud/src/main/kotlin/com/knf/dev/demo/kotlinspringjpamustachecrud/repository/UserRepository.kt
package com.knf.dev.demo.kotlinspringjpamustachecrud.repository

import com.knf.dev.demo.kotlinspringjpamustachecrud.model.User
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface UserRepository : CrudRepository<User?, Long?>